khan_linter.go

```diff
@@ -0,0 +1,6 @@
+package khan_linter
+
+import (
```

Comment:
Can you add a comment why this file is needed?

Reply:
Sure. revert linter.py in this commit, and will send diff of it in another commit.
Yes, I add comments about why we need this empty go module.